project/Version.scala

```diff
@@ -12,10 +12,10 @@ import sbt.Keys._
 object Version {
 
   def versionSettings: Seq[Setting[_]] = inConfig(Compile)(Seq(
-    resourceGenerators <+= generateVersion(resourceManaged, _ / "version.conf",
+    resourceGenerators += generateVersion(resourceManaged, _ / "version.conf",
```

Comment:
Seems like it works on my computer (to create version files). Will carefully check it again tomorrow!